README.md

```diff
@@ -1,3 +1,18 @@
+# Copyright (c) 2014 - 2015 YCSB contributors. All rights reserved.
```

Comment:
this renders as headlines. instead, surround the headers for markdown files with &lt;!-- and --&gt;

e.g.

```
<!--
Copyright (c) 2014 - 2015 YCSB contributors
-->
```

Incidentally, the top level README must be older than 2014, right? Is it basically :copyright: 2010 Yahoo! and :copyright: 2010 - 2015 YCSB contributors?